docs/src/pages/components/data-grid/filtering/ColumnTypeFilteringGrid.js

```diff
@@ -57,7 +57,7 @@ export default function ColumnTypeFilteringGrid() {
         }}
         state={{
           preferencePanel: {
-            open: true,
+            open: false,
```

Comment:
if you set open to false. you can remove the full state prop

Reply:
Ok, I assumed that `openedPanelValue` would still be valuable but I didn't test it. I can remove the whole `state` prop if it has no effect.

Reply:
yes because if the panel is not open then it doesn't matter which panel should be opened

Reply:
OK, I'm removing it then. I actually like that we don't use this private `state` prop in a demo anymore, better 👌